contracts/libraries/LibDataTypes.sol

```diff
@@ -12,13 +12,15 @@ library LibDataTypes {
      * @param PROXY Creates a dedicated caller (msg.sender) to be used when executing the task. {See ProxyModule.sol}
      * @param SINGLE_EXEC Task is cancelled after one execution. {See SingleExecModule.sol}
      * @param WEB3_FUNCTION Use off-chain condition & input data for execution. {See Web3FunctionModule.sol}
+     * @param TRIGGER Repeated execution of task ata a specified timing and interval or cron. {See TriggerModule.sol}
      */
     enum Module {
         RESOLVER,
         TIME,
         PROXY,
         SINGLE_EXEC,
-        WEB3_FUNCTION
+        WEB3_FUNCTION,
+        TRIGGER
```

Comment:
Will `TRIGGER` module be strictly necessary when `WEB3_FUNCTION` module is enabled? If so, we might need to enforce it here: https://github.com/gelatodigital/automate/blob/0fab4305f092f8ff2ba675e6cae87591926039f5/contracts/libraries/LibTaskModule.sol#L287-L292